### PR TITLE
ua-ch updates

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/280482338

Changes proposed in this pull request:

- Update User-Agent reduction docs for scaled availability 
- some copy edits
- staged: https://pr-6179-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/user-agent/

### CODE REVIEW

File: site/en/docs/privacy-sandbox/user-agent/index.md [hunk 1]
```diff
@@ -132,7 +130,7 @@ Critical-CH: Device-Memory
 ```
 
 If, after processing the `Accept-CH header`, the client would send a critical
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/user-agent/index.md:132]: Something is confusing in this sentence

File: site/en/docs/privacy-sandbox/user-agent/index.md [hunk 2]
```diff
@@ -181,7 +179,7 @@ increasing in complexity.
 
 Scaled availability for User-Agent reduction means the fully reduced UA string
 shipped on all Chrome devices. Reduction is planned to begin with a Chrome
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/user-agent/index.md:181]: Reduction is planned to begin with a ...
Is this still correct?

File: site/en/docs/privacy-sandbox/user-agent/index.md [hunk 3]
```diff
@@ -131,8 +129,7 @@ Vary: Device-Memory, Viewport-Width
 Critical-CH: Device-Memory
 ```
 
-If, after processing the `Accept-CH header`, the client would send a critical
-hint, the client retries the request.
+If the original request included a `Critical-CH`, after processing the `Accept-CH header`, the  client retries the request.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/user-agent/index.md:132]: Is this correct?

Comment (heyawhite) [reply site/en/docs/privacy-sandbox/user-agent/index.md:132]:
```suggestion
If the browser requires a critical hint (`Critical-CH`) to properly render the webpage, then the server can ask for this additional information with the `Accept-CH` header. Then, the browser can send a new request for the page, including the critical hint.
```

I think this suggestion makes a little more sense.

**Review by heyawhite — APPROVED** [review]
One suggestion, per your question

File: site/en/docs/privacy-sandbox/user-agent/index.md (hunk 2 above)
Comment (heyawhite) [reply site/en/docs/privacy-sandbox/user-agent/index.md]: Should be "Reduction began with..."

**Pull request merged**